sys/include/net/gnrc/netif/hdr.h

```diff
@@ -46,6 +46,19 @@ extern "C" {
  */
 #define GNRC_NETIF_HDR_L2ADDR_PRINT_LEN (GNRC_NETIF_HDR_L2ADDR_MAX_LEN * 3)
 
+/**
+ * @brief   Special value to indicate that no RSSI value is present
+ *
+ * See @ref gnrc_netif_hdr_t::rssi
+ */
+#define GNRC_NETIF_HDR_NO_RSSI          (INT16_MIN)
+/**
+ * @brief   Special value to indicate that no LQI value is present
+ *
+ * See @ref gnrc_netif_hdr_t::lqi
+ */
+#define GNRC_NETIF_HDR_NO_LQI           (INT8_MIN)
```

Comment:
This gives `0x80`/127 which is a reasonable LQI value.
255 is the best possible LQI, a LQI of 0 would be worst, not sure if such packets could still be observed.

Reply:
I though I read `int8_t`. There doesn't seem to be a standard for LQI, other transceivers give valies from 0 to 100. There -128 (if it would have been signed) would be clearly out of range.

I guess than `UINT8_MIN` is the best value and drivers would habe to round up zero one :-(

Reply:
I also fixed the the PR and commit description right away.